src/styles/Blocks/star.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-size: auto;
+    background-position: center;
+    background-repeat: no-repeat;
+    width: 16px;
+    height: 16px;
+    border-radius: 0.5px 0 0;
```

Comment:
`border-radius: 1px 0 0;`

It does not exists half values for pixels, just integers